includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+class DesignSystemGlobalNavigationModel extends WikiaModel {
+	const DEFAULT_LANG = 'en';
+
+	private $hrefs = [
+		'default' => [
+			'fandom-logo' => 'http://fandom.wikia.com',
+			'games' => 'http://fandom.wikia.com/games',
+			'movies' => 'http://fandom.wikia.com/movies',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "movies" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)